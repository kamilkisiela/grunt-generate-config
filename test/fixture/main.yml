app:
    url: %app_url%
    
api:
    url: %api_url%
    #hide credentials
    credentials:
        username: app
        password: "N&F3BUAlPrhLi9@BRmqg"

sync:
    output: front/vendor/sync-version/sync-version.js