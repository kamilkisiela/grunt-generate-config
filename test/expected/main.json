{"app":{"url":"http://app.wieldo","test":123},"api":{"url":"http://api.wieldo","credentials":{"username":"app","password":"N&F3BUAlPrhLi9@BRmqg"}},"sync":{"output":"front/vendor/sync-version/sync-version.js"}}